classes/table/acknowledged_notice.php

```diff
@@ -61,14 +61,14 @@ class acknowledged_notice extends table_sql implements renderable {
      *
      * @param string $uniqueid id of the table.
      * @param \moodle_url $url base url.
+     * @param int $noticeid notice id.
      * @param array $filters  filter.
      * @param string $download download file format.
      * @param int $page current page.
      * @param int $perpage  number of record per page.
-     * @param int $noticeid notice id.
      */
-    public function __construct($uniqueid, \moodle_url $url, $filters = [],
-                                $download = '', $page = 0, $perpage = 20, $noticeid) {
+    public function __construct($uniqueid, \moodle_url $url, $noticeid, $filters = [],
```

Comment:
any specific reason why we need to change the order here and below?

Reply:
By PHP 8.0 - optionnal parameters should always be _after_ required ones